content/chainguard/vms/compliance-features.md

```diff
@@ -0,0 +1,97 @@
+
+## **STIG Hardening**
+
+Chainguard VMs provide variants hardened to DISA Security Technical Implementation Guide (STIG) requirements used across U.S. federal and defense environments.
+
+* 2000+ relevant system controls are pre-configured and validated in the image, not delivered as a checklist that the customer must implement.
+* Images are designed as ready-to-use replacements for existing Linux OS images, enabling STIG-compliant baselines without re-architecting existing infrastructure.
+
+Chainguard can also provide SCAP scan outputs aligned with STIG requirements, helping teams demonstrate compliance with control requirements during audits.
+
+## **CIS Benchmark Compliance**
+
+For organizations standardizing on CIS controls, Chainguard offers images hardened to **CIS Level 1** benchmarks.
+
+* Images are available in **CIS L1 hardened variants** for Chainguard VMs.
+* Chainguard VMs use a **hybrid baseline combining CIS L1 benchmarks with STIG requirements and industry-recognized secure defaults** to provide defense-in-depth hardening.
+
+This allows security and GRC (Governance, Risk, and Compliance) teams to map infrastructure posture to both internal CIS-based policies and external STIG-based requirements without maintaining parallel baselines.
+
+## **Secure Boot**
+
+All Chainguard VM images support **Secure Boot enabled by default** across:
+
+* AWS
+* Microsoft Azure
+* Google Cloud Platform
+* On-premises platforms supporting Secure Boot
+
+Secure Boot ensures only cryptographically signed and trusted components participate in the boot chain, preventing tampering with early-boot components such as the bootloader and kernel.
+
+## **Compliance Evidence and Reporting**
+
+Chainguard VMs are designed to simplify generation of compliance artifacts often required in audits, ATO processes, and customer security reviews.
+
+Available artifacts include:
+
+* FIPS module documentation and integration details, including OpenSSL certificate documentation.
+* SCAP/OpenSCAP scan results for STIG and CIS benchmarks.
+* POA\&M-ready information (Plans of Action and Milestones) to support risk tracking and remediation planning.
+
+By shipping this evidence with the images, Chainguard significantly shortens the time required to build audit packages and meet regulatory reporting needs.
+
+## **Vulnerability Management and Lifecycle**
+
+Chainguard VMs are built and maintained with an explicit **CVE remediation SLA**:
+
+* **Critical CVEs**: patched within **7 days**
+* **High, medium, and low CVEs**: patched within **14 days**
+
+Operational implications:
+
+* Images are updated regularly and made available to customers within these SLA windows.
+* The minimal, hardened footprint reduces the volume of installed software and therefore, the attack surface and CVE count that teams must manage.
```

Comment:
```suggestion
Chainguard VM images are updated regularly and made available to customers within these SLA windows, leaving them with a minimal, hardened footprint which reduces the volume of installed software and minimizes the attack surface. This leaves customers with a much smaller and more manageable CVE count.
```

Trying to make these bullets a little more straightforward. LMK if this suggestion causes any important meaning to be lost.